ntp.go

```diff
@@ -347,8 +342,8 @@ func getTime(host string, opt QueryOptions) (*msg, ntpTime, error) {
 		return nil, 0, err
 	}
 
-	dt41 := time.Since(xmt_time) // uses monotonic clock @ Go 1.9+
-	dst := toNtpTime(xmt_time.Add(dt41))
+	dt41 := time.Since(xmtTime) // uses monotonic clock @ Go 1.9+
```

Comment:
Now that I know what you mean by `dt41` I understand, but I still think it's a little cryptic, since the T1..T4 terminology is only used in a couple helper functions later in the code.  Can we just call it `diff` or `delta` or something like that?

Reply:
> since the T1..T4 terminology is only used in a

Yeah. Sorry, I spent too much time reading ntp.org sources, it's quite popular notation there :)